packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -291,61 +323,10 @@ const numberWithUnitCellView = NumberWithUnitCellView.compose({
 });
 ```
 
-Here we have a cell visualized in different ways based on custom logic:
+#### Exposing cell view events externally
```

Comment:
I commented out this part of the spec because it talks about a pattern for responding to events from your cell view instance that is actively being discussed in #1153 
Responding to anchor tag clicks is exactly the use-case this part of the spec was trying to consider. As part of #1153 we should update this part of the spec for event handling column view @m-akinc

Reply:
Aded https://github.com/ni/nimble/pull/1153/files#r1161898729